src/rox.rs

```diff
@@ -1,16 +1,42 @@
 use scanner::Scanner;
+use std::collections::HashMap;
 use std::fs::File;
 use std::io;
 use std::io::prelude::*;
 use std::io::{BufRead, Write};
 use std::process;
+use token_type::TokenType;
 
 #[derive(Debug)]
 pub struct Rox {
     pub had_error: bool,
+    pub keywords: HashMap<String, TokenType>,
 }
 
 impl Rox {
+    pub fn new(had_error: bool) -> Rox {
+        let mut keywords = HashMap::new();
+        keywords.insert("and".to_string(), TokenType::And);
```

Comment:
Is this the best place to initialize Rox's keywords? A constructor _feels_ like the right place, but I almost expect to be told otherwise.